app/code/Magento/InventoryConfiguration/Model/GetStockItemConfiguration.php

```diff
@@ -97,7 +116,7 @@ private function getLegacyStockItem(string $sku): StockItemInterface
         try {
             $productId = $this->getProductIdsBySkus->execute([$sku])[$sku];
         } catch (NoSuchEntityException $skuNotFoundInCatalog) {
-            $stockItem = \Magento\Framework\App\ObjectManager::getInstance()->create(StockItemInterface::class);
+            $stockItem = ObjectManager::getInstance()->create(StockItemInterface::class);
```

Comment:
Why do we use ObjectManager manager directly instead of construct dependency? Or it is a temporal solution?